proto/dwtools/abase/l0.test/Long.test.s

```diff
@@ -11907,17 +11935,45 @@ function longAllAreRepeated( test )
   var expected = true;
   test.identical( got, expected );
 
+  test.case = 'unroll, single number with evaluator';
+  var evaluator = ( e ) => e;
+  var src = _.unrollMake( [ 1 ] );
+  var got = _.longAllAreRepeated( src, evaluator );
+  var expected = false;
+  test.identical( got, expected );
+
+  test.case = 'unroll, single value with evaluator';
+  var evaluator = ( e ) => e;
+  var src = _.unrollMake( [ { val: 1 } ] );
+  var got = _.longAllAreRepeated( src, evaluator );
+  var expected = false;
+  test.identical( got, expected );
+
+  test.case = 'unroll, some numbers are identical with evaluator';
+  var evaluator = ( e ) => _.numberIs( e );
+  var src = _.unrollMake( [ 1, 2, 2 ] );
+  var got = _.longAllAreRepeated( src, evaluator );
+  var expected = true;
+  test.identical( got, expected );
+
   test.case = 'unroll, some values are identical with evaluator';
   var evaluator = ( e ) => e.val;
   var src = _.unrollMake( [ { val: 1 }, { val: 1 }, { val: 3 } ] );
   var got = _.longAllAreRepeated( src, evaluator );
   var expected = false;
   test.identical( got, expected );
 
+  test.case = 'unroll, complex data of array with evaluator';
+  var evaluator = ( e ) => e[ 0 ];
+  var src = _.unrollMake( [ [ 1, 0 ], [ 1, 1 ], [ 1, 2 ] ] );
+  var got = _.longAllAreRepeated( src, evaluator );
+  var expected = true;
+  test.identical( got, expected );
+
   /* With equalizer */
```

Comment:
Equalizer and evaluator are callbacks. So, if some test case exists in test group `with evaluator`, then similar test case should be in test group `with equalizer`. 

Please, check test cases and add similar test cases to group `with equalizer`.

Check routines `longAnyAreRepeated` and `longNoneAreRepeated`, please.